intermine/bluegenes
# Google Analytics

To test: 
- set a google tracking ID in your `config/dev/config.edn` (or prod in place of dev)
- navigate to various pages. You should be able to see the analytics tracking in real-time in your google analytics account.

if there is no google analytics tracking id, it'll print to the console saying there's no tracking but otherwise operate smoothly.
